cid/main.py

```diff
@@ -33,8 +33,14 @@ def read_root() -> dict:
 
 
 @app.get("/aws")
-def all_aws_images(db: Session = Depends(get_db)) -> list:  # noqa: B008
-    result = db.query(AwsImage).order_by(AwsImage.creationDate.desc()).all()
+def all_aws_images(
```

Comment:
@major We could theoretically merge the "/aws/image/{image_id}" endpoint into this by adding an aditional "ami" parameter. WDYT?

Reply:
I like that idea! 👍